From 8d09ceddef0a4056d3ad935e12c18d47022b6261 Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Thu, 21 Oct 2021 13:01:00 +0100
Subject: [PATCH] phy: renesas: phy-rcar-gen3-usb2: Add RZ/G2L support

SoC ID r9a07g044l2 is missing from the soc_no_adp_ctrl list,
add it.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 drivers/phy/renesas/phy-rcar-gen3-usb2.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/phy/renesas/phy-rcar-gen3-usb2.c b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
index 48fbe4370175..767e391a8396 100644
--- a/drivers/phy/renesas/phy-rcar-gen3-usb2.c
+++ b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
@@ -535,6 +535,10 @@ static irqreturn_t rcar_gen3_phy_usb2_irq(int irq, void *_ch)
 }
 
 static const struct soc_device_attribute soc_no_adp_ctrl[] = {
+	{
+		.soc_id = "r9a07g044l2",
+		.data = (void *)USB2_NO_ADP_CTRL,
+	},
 	{
 		.soc_id = "r9a07g044l",
 		.data = (void *)USB2_NO_ADP_CTRL,
-- 
2.25.1

